docs/form-controls/form-section-header.md

```diff
@@ -19,26 +19,27 @@ eleventyNavigation:
 - Put the form section header and its related form fields into the same `.content-container`.
 - Put `role="group"` onto the container which holds the form section.
 - Be sure to consult [Typography and Headings](/foundation/typography/#headings) to ensure you’re using the appropriate heading level.
-
-**Note:** The content container, row, and column are only included to show how the Form Section Header will look. If you already have those in place, you only need the element with “Form Section” as the text.
+- Remove the `role="presentation"` from the `<h2>` markup when you paste it into your project.
```

Comment:
Quite confused on the purpose of the code snippet here. Can we demo where the role=group is? Can we also just remove the role=presentation in the snippet?

Reply:
@frogpad7 pushed up a change. No, i don’t want this sample header to cause an extra heading for SRs

Reply:
@frogpad7 just this one rn